.github/workflows/golangci_lint.yml

```
@@ -36,4 +39,5 @@ jobs:
         uses: golangci/golangci-lint-action@v6
         with:
           version: latest
-          only-new-issues: true
+          # When is scheduled, show all issues
+          only-new-issues: ${{ github.event_name != 'schedule' }}
```

I've been thinking about how we can be notified about when the scheduled version of the action runs. I found the [`Failed Build Issue` Action](https://github.com/kubearchive/kubearchive/pull/922/files) which can create an issue for us if the `golangci-lint` fails. The action needs `Read and write permissions`

```suggestion
          only-new-issues: ${{ github.event_name != 'schedule' }}
      - name: Notify golangci-lint failed
        uses: jayqi/failed-build-issue-action@v1
        if: failure() && github.event_name != 'schedule' && github.repository == 'kubearchive/kubearchive'
        with:
          github-token: ${{secrets.GITHUB_TOKEN}}
          title-template: "golangci-lint action failed"
          # by default the action will create a comment on the newest issue with this label so it should get its own label to use
          label-name: "golangci-lint"
```

---

I added something similar.